guides/common/assembly_configuring-capsule-custom-server-certificate.adoc

```diff
@@ -9,9 +9,9 @@ If you configure {ProjectServer} to use a custom SSL certificate, you must also
 
 To configure your {SmartProxyServer} with a custom certificate, complete the following procedures on each {SmartProxyServer}:
 
-. xref:creating-a-custom-certificate_capsule[]
-. xref:deploying-a-custom-ssl-certificate-to-capsule-server_capsule[]
-. xref:deploying-a-custom-ssl-certificate-to-hosts_capsule[]
+. xref:creating-a-custom-certificate_{smart-proxy-context}[]
+. xref:deploying-a-custom-ssl-certificate-to-capsule-server_{smart-proxy-context}[]
+. xref:deploying-a-custom-ssl-certificate-to-hosts_{smart-proxy-context}[]
```

Comment:
Just a heads up as I haven't tried to build this - I saw that variables cannot be used in some instances like in anchors. Doublecheck this renders correctly.

Reply:
Before I committed these changes I tested this throughout all versions of the guide and this works! Thanks.